BaseFilter/sections/adservers.txt

```diff
@@ -9,6 +9,7 @@
 !
 !
 !
+||evpdcnksmf.com^
```

Comment:
Where does this request appear?

Reply:
@jellizaveta I can't reproduce now, maybe blocking `||groodsuy.com^` is more useful?

Reply:
which script makes a request to this domain?